.github/workflows/pr-subtree-push.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       with:
         subtree: apollo-ios-pagination
         remote: git@github.com:apollographql/apollo-ios-pagination.git
-        target-branch: hs/complete-rewrite
+        target-branch: main
```

Comment:
For this to be true does it require that the ['Complete Rewrite' PR](https://github.com/apollographql/apollo-ios-pagination/pull/8) be merged into `main` first?

Reply:
Yes -- Let me merge that one first.

Reply:
@calvincestari merged 👍🏽